packages/core/src/scuttle.js

```diff
@@ -22,6 +22,14 @@ const {
   defineProperty,
 } = Object
 
+const avoidForLavaMoatCompatibility = (function(cg, g) {
+  return Object.getOwnPropertyNames(cg)
+    // skip all intrinsics that a Compartment already grants
+    .filter(a => cg[a] === g[a])
+    // support LM,SES exported APIs and polyfills that LM counts on
+    .concat(['Compartment', 'Error', 'globalThis'])
+}(new Compartment().globalThis, globalThis))
```

Comment:
neat. We don't need an API for listing intrinsics after all

Reply:
@boneskull let's also ponder how that relates to us giving Endo a subclassed compartment constructor